collect_app/src/main/java/org/odk/collect/android/database/ActivityLogger.java

```diff
@@ -109,7 +109,6 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
 
     private final boolean loggingEnabled;
     private final String deviceId;
-    private DatabaseHelper databaseHelper;
```

Comment:
@grzesiek2010 @shobhitagarwal1612 Ok, this seems to be the commit that broke the singleton pattern and caused resource leaks. The point stands -- we/I need to be super careful when introducing and reviewing these.